### ISSUE TITLE
STA/OPR 9750 mussakone MiniProject #03

### ISSUE THREAD

**Issue by mussakone** [mussakone/STA9750-2024-FALL#4]
Hi @michaelweylandt!

https://mussakone.github.io/STA9750-2024-FALL/mp03.html

**Comment by michaelweylandt** [comment]
Thanks @mussakone !

Peer Feedback:

- @Malexandra-Jerez
- @aaronkcooper
- @cliu98
- @laurencardieri

It is now time for the peer feedback round for Mini-Project 03. Please
review @mussakone's submission for this mini-project and provide peer feedback.

Using the rubric at https://michael-weylandt.com/STA9750/miniprojects/mini03.html#rubric,
please grade their submission out of a total of 50 points. Note that this
rubric is slightly different than that used for Mini-Projects 01 and 02.

For each of the five categories, please give them a separate score and provide
a total (sum) score across the entire assignment. Feel free to assign extra
credit if you feel it is warranted (following the rubric).

If you give a score of less than 5 for any category, please provide a suggestion
for improvement. (You can also give suggestions for any element they did well -
more feedback is always great!)

As you go through this peer feedback exercise, think about what you particularly
like about this submission and how you can incorporate that approach in your future
work. If something is particularly insightful or creative, give some kudos!

Evaluators: This should take you around 15 minutes per peer feedback. You are
not required to engage in substantial back-and-forth with @mussakone, but you are of course
welcome to initiate a discussion.

@mussakone: please engage fully with your peers. They are here to help you!

Submission URL should be: https://mussakone.github.io/STA9750-2024-FALL/mp03.html

Feel free to link to other repos, the course documentation, or other useful
examples.

Thanks!
@michaelweylandt

CC: @charles-ramirez

**Comment by cliu98** [comment]
Hi @mussakone 

Here's the breakdown for my feedback:

Written Communication: 7/10
Your report can benefit from more writing overall. Task 3 specifically includes no insights into your findings but just data outputs. Bringing your report together with writing can make it flow better to where you ultimate end with your fact check. 

Project Skeleton: 9/10
Task 7 would have benefitted if you included your data analysis code with visuals or tables with exact numbers to support your claims made. 

Formatting & Display: 7/10
When you execute R code in Markdown, you can hide warnings, messages, etc that create a lot of unnecessary outputs in the report. To get the point across easier to the reader, only outputting the results you want us to understand would be cleaner. This can be done in the portion ```{R, results = "hide", message = FALSE, warning = FALSE}.

The tables can be better formatted with the column names. They're left as variable names but this can be easily updated with a `rename` function for the table output. 

Code Quality: 10/10

Data Preparation: 9/10 
Since we want this report to be reproduceable in nature, I would include the download links for the MIT Election Data Science Lab datafiles, otherwise anyone looking at it won't know where to get those files you are referencing since there is no automated download for it.

Overall: 42/50
Your report is spot on in terms of what was required in the technical aspect. My only critiques would be to clean up the formatting and code outputs, and include more written insights for your work. You can let the data speak for itself, but combining it with a written portion will only make your findings more engaging for the readers.

**Comment by aaronkcooper** [comment]
Hi @mussakone,
Here is my breakdown of feedback:
**Written Communication: 6/10**
The written report is very hard to follow and offers little explanation behind most of the tasks that you are completing. Task questions are copy pasted in big bold headers, overall there is little analysis or flow outside of task 7.

**Project Skeleton: 8/10**
Task 7 is not supported at all by analysis done and shown using the dataset. I can say anything I want to, but without data or visualizations of the data supporting my claims, such claims are meaningless. 

**Formatting and Display: 6/10**
Graphs are done well overall. I think the report really could have benefited from more displays over tables, or at least more concise tables at that. For the future, please hide the output and warnings from R in your markdown. It takes up far too much space and is really distracting when looking at the report. 

**Code Quality: 9/10**
The automated downloading of the data is done in a big for loop in the console, wouldn't it make sense to contain this in a function?

**Data Preparation: 10/10**

**Total Score: 39/50**

**Comment by laurencardieri** [comment]
Hi Mussakone! My feedback is as follows:

Written Communication: 6/10. Some punctuation marks and spaces were missing. There also was not many written explanations of your findings or what you were doing, especially in task 3. Thus, I would recommend including a lot more written sections and proofreading them. Other than that, I believe you did a great job! 

Project Skeleton: 8/10. I believe you did most of what the professor required well for this mini project. However, you did forget to add the citations section and did not support Task 7 by any data. I also would have loved to see more graphs, instead of tables. 

Formatting and Display: 6/10. There were a lot of code feedback given from your coding in the final report, such as "column specification". This made the report harder to follow for the viewer. There was also an anticipated heading that did not become a heading due to a missing space. Your maps did not include Alaska or Hawaii, and only went from 2004 instead of 1976. Other than that, I believe your project was formatted well. 

Code Quality: 10/10. I did not find any errors within your code! Great work! 

Data Preparation: 10/10. Your data preparation was done extremely well. You were able to automate it and restrict the code from redownloading if it was already downloaded. Great work! 

Total Score: 40/50

Great Work!
